### ISSUE TITLE
bug: No code suggestions appear

### ISSUE THREAD

**Issue by nullromo** [yetone/avante.nvim#1011]
### Describe the bug

I used `:AvanteAsk` to tell the AI to indent my file. It says `**Generation complete!** Please review the code suggestions above.`, but there are no code suggestions above.

### To reproduce

1. Open vim
2. `:AvanteAsk`
3. Ask it to indent the file
4. The output looks like this:
```
- Datetime: 2024-12-27 14:16:01

- Model: claude/claude-3-5-sonnet-20241022

- Selected files:
  - plugins/avante.lua

> indent the file by 12 spaces

I'll add 12 spaces to the beginning of every line while maintaining the existing indentation structure.

Filepath: plugins/avante.lua

**Generation complete!** Please review the code suggestions above.
```
5. Now what?

### Expected behavior

I'm not sure what to expect, but it basically did nothing at all, so I don't understand what I'm supposed to do from here.

### Installation method

Use lazy.nvim with no changes from the recommended options.

<details>

```lua
{
    'yetone/avante.nvim',
    event = 'VeryLazy',
    lazy = false,
    version = false,
    opts = {},
    build = 'make',
    dependencies = {
        'stevearc/dressing.nvim',
        'nvim-lua/plenary.nvim',
        'MunifTanjim/nui.nvim',
        'hrsh7th/nvim-cmp',
        'nvim-tree/nvim-web-devicons',
        'zbirenbaum/copilot.lua',
        {
            'HakonHarnes/img-clip.nvim',
            event = 'VeryLazy',
            opts = {
                default = {
                    embed_image_as_base64 = false,
                    prompt_for_file_name = false,
                    drag_and_drop = { insert_mode = true },
                    use_absolute_path = true,
                },
            },
        },
        {
            'MeanderingProgrammer/render-markdown.nvim',
            opts = { file_types = { 'markdown', 'Avante' } },
            ft = { 'markdown', 'Avante' },
        },
    },
}
```

</details>

### Environment

Version: NVIM v0.11.0-dev
Platform: WSL

**Comment by mark-omarov** [comment]
Make sure you have credit balance, check Anthropic Console. Your API key won't work otherwise.
Could have note in docs or error/warning in the chat view.

**Comment by svenbergner** [comment]
I have the same problem. It works correct when I use CopilotChat.nvim. So I think it's not a problem with my credit balance.

**Comment by nullromo** [comment]
This was my process, and I'm sure I'm not the only one in this situation

1. I am a software developer
1. People are talking about AI; I have no idea how to use it
1. Friend suggests this plugin
1. Cool, maybe I'll learn how to use AI at work
1. Add the plugin to my lazy.nvim config, try it out, and it asks for something called `ANTHROPIC_API_KEY`
1. No idea what that is
1. Google it
1. Oh, looks like I need an account
1. Create an account
1. Find the key and add it to my .bashrc
1. Try the plugin again
1. I get a the response posted above
1. > "make sure you have credit balance"
1. No idea what that is
1. Google it
1. Apparently you have to pay for this ...?
1. Apparently I have $4.90 credit in my account
1. Seems like maybe I got $5 free to start and I've used 10¢ of it ...?

And here we are. From what I can tell, "I'll add 12 spaces to the beginning of every line while maintaining the existing indentation structure." came from the AI, so that indicates to me that it's at least able to connect to the AI backend server and get some kind of response. And presumably that's what used up the 10¢ credit.

**Comment by wouterschiedam** [comment]
I'm experiencing the same behavior where it generates text. But then keeps "Searching..." when it tries to write a code block until at some point it writes generation complete.

**Comment by gopz** [comment]
I just did a fresh install, I think I got everything right with the installation (used default) and API key. Whenever I submit a prompt/question I get something like this:

```
- Datetime: 2024-12-30 13:55:25

- Model: claude/claude-3-5-sonnet-20241022

- Selected files:
  - config/tag_maps/common.yml

> See if any tag codes are repeated in this file



**Generation complete!** Please review the code suggestions above.
```

Basically, no indication it did anything and no error message. If I change the provider to OpenAPI, it seems to work.

**Comment by bernoussama** [comment]
same
![image](https://github.com/user-attachments/assets/f5c2d0b2-6de0-40ec-89a8-46524dcde1d9)

**Comment by synic** [comment]
Yeah avante seems pretty busted lately. Incomplete responses, broken diffs, etc. Getting almost unusable. Strange because it seemed just fine not too long ago

**Comment by bcanvural** [comment]
This has been a new development for me. SInce last few updates the prompts are often cut short. I %100 have enough credit  with gpt4

**Comment by fjarachp** [comment]
I had the same issue, i was using https://www.lazyvim.org/ and i switched to kickstart.nvim and it worked. LazyVim does a lot of thing behind scenes that sometimes it has conflicts with certain plugins

**Comment by JoshJarabek7** [comment]
I'm facing this bug as well on the latest version.

`return {
  {
    "yetone/avante.nvim",
    event = "VeryLazy",
    lazy = false,
    version = "*", -- Set this to "*" to always pull the latest release version, or set it to false to update to the latest code changes.
    opts = {
      request_timeout = 86400000,
      provider = "openai",
      openai = {
        endpoint = "https://api.openai.com/v1",
        model = "o1-2024-12-17",
        temperature = 0.7,
        stream = false,
        max_completion_tokens = 100000,
      },
      history = {
        max_tokens = 140000,
      },
    },
    -- if you want to build from source then do `make BUILD_FROM_SOURCE=true`
    build = "make",
    -- build = "powershell -ExecutionPolicy Bypass -File Build.ps1 -BuildFromSource false" -- for windows
    dependencies = {
      "stevearc/dressing.nvim",
      "nvim-lua/plenary.nvim",
      "MunifTanjim/nui.nvim",
      --- The below dependencies are optional,
      "echasnovski/mini.pick", -- for file_selector provider mini.pick
      "nvim-telescope/telescope.nvim", -- for file_selector provider telescope
      "hrsh7th/nvim-cmp", -- autocompletion for avante commands and mentions
      "ibhagwan/fzf-lua", -- for file_selector provider fzf
      "nvim-tree/nvim-web-devicons", -- or echasnovski/mini.icons
      "zbirenbaum/copilot.lua", -- for providers='copilot'
      {
        -- support for image pasting
        "HakonHarnes/img-clip.nvim",
        event = "VeryLazy",
        opts = {
          -- recommended settings
          default = {
            embed_image_as_base64 = false,
            prompt_for_file_name = false,
            drag_and_drop = {
              insert_mode = true,
            },
            -- required for Windows users
            use_absolute_path = true,
          },
        },
      },
      {
        -- Make sure to set this up properly if you have lazy=true
        "MeanderingProgrammer/render-markdown.nvim",
        opts = {
          file_types = { "markdown", "Avante" },
        },
        ft = { "markdown", "Avante" },
      },
    },
  },
}
`

Neovim: 0.10.3
MacOS: 15.2

**Comment by tlelson** [comment]
Is this fixed tho?
I'm on the latest usable commit (halfway between `v0.0.22` and `v0.0.23` @ `d75094b`) and I get this with every model.
